### PR TITLE
Issue 236: New command for generating Gradle projects with Kotlin DSL

### LINKED ISSUE #236: i can't choose kotlin in gradle DSL
hi.
As the title suggests, I can't choice kotlin of gradle DSL in this plugin.
I'm not good at English, so it's difficult to write many words.
I would like you to check it.

### PR DESCRIPTION
**Pull request by brunovieira97**
New command `spring.initializr.gradle-project-kotlin` for generating Gradle projects with Kotlin DSL.

Closes #236

### CODE REVIEW

**Comment by brunovieira97** [comment]
@jdneo @testforstephen this PR has been waiting for a review for a while, could you take a look?

**Review by testforstephen — APPROVED** [review]
LGTM. thanks for contribution.

File: src/extension.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
 } from "vscode-extension-telemetry-wrapper";
 import { AddStartersHandler, GenerateProjectHandler } from "./handler";
 import { getTargetPomXml, loadPackageInfo } from "./Utils";
+import { ProjectType } from "./model";
```
Comment (testforstephen) [inline src/extension.ts:15]: Linting failed. 

`ERROR: (ordered-imports) src/extension.ts[15, 1]: Import sources within a group must be alphabetized.`

**Review by testforstephen — CHANGES_REQUESTED** [review]

File: src/extension.ts (hunk 1 above)
Comment (brunovieira97) [reply src/extension.ts:14]: Sorry for that @testforstephen! Fixed it.

**Review by testforstephen — APPROVED** [review]

**Pull request merged**